src/main.c

```diff
@@ -173,7 +173,7 @@ int main(void) {
     }
 
     if (sys_flow_cntrl_u.bits.nfc_off == false) {
-      nfc_deselect_card();
+      // nfc_deselect_card();
```

Comment:
probably not needed to comment this out ATM

Reply:
fixed